decepticonlp/metrics/char_metrics.py

```diff
@@ -0,0 +1,126 @@
+import numpy as np
+
+
+def levenshtein(text1, text2, normalize="none"):
+    """
+    Calculate Levenshtein Distance using dynamic programming optimized with (np)
+    DP - O(m*n) complexity - Recursive approach - O(3^m)
+
+    Example:
+    from perturb import levenshtein
+    print(levenshtein("Hey","HEY"))
+    2.0
+
+    #Normalize Levenshtein Distance - Total strategy
+    print(levenshtein("Hey", "HEY", normalize="sum"))
+    0.33333
+
+    #Normalize LCS - Max Strategy
+    print(levenshtein("HeyS", "HEY", normalize="lcs"))
+    0.75
+
+
+    :params
+    :text1, text2 - Both the inputs
+    :normalize - Pass "none" for getting the raw distance
+               - Pass "sum" for getting Normalized Levenshtein Distance
+               - Pass "lcs" for getting #Normalize LCS
```

Comment:
This is not the correct format I think.
It should be 
```
        :param a: First Letter
        :param B: Second Letter
        :type a: char
        :type B: char 
        :returns: z based on the Q table 
        :rtype: char ```

Reply:
Nice I will update the changes